dialog/file.go

```diff
@@ -249,23 +301,55 @@ func showFileDialog(save bool, callback interface{}, parent fyne.Window) {
 	d.win.Resize(size)
 
 	d.win.Show()
+	return d
 }
 
-// ShowFileOpen shows a file dialog allowing the user to choose a file to open.
-// The dialog will appear over the window specified.
-func ShowFileOpen(callback func(fyne.FileReadCloser, error), parent fyne.Window) {
```

Comment:
Why not keep the ShowFileOpen as a shortcut to New...().Show() like we discussed?